web/client/reducers/dimension.js

```diff
@@ -49,6 +50,10 @@ module.exports = (state = {}, action) => {
             }
             return set(`currentTime`, action.time, state);
         }
+        case REMOVE_NODE: {
+            const newData = mapValues(state.data, (o) => pickBy(o, (values, keys) => keys !== action.node));
```

Comment:
```
set(`data[${action.node}]`, undefined, state) 
```
 - Is more clear and readable
 - works even if state or state.data is undefined
Why did you have to do this strange copy? 
Notice that the set is from ImmutableUtils.